modules/smart-agent_cassandra-nodetool/conf/00-status.yaml

```diff
@@ -0,0 +1,18 @@
+## Example
```

Comment:
You can drop this line while it is not an example.

also there is no explicit rule about that but in general we use the `00` for the heartbeat so if it is ok for you please start count at 1 ?